pkgs/tools/audio/darkice/default.nix

```diff
@@ -5,11 +5,11 @@
 
 stdenv.mkDerivation rec {
   pname = "darkice";
-  version = "1.3";
+  version = "1.4";
 
   src = fetchurl {
-    url = "mirror://sourceforge/darkice/${version}/darkice-${version}.tar.gz";
-    sha256 = "1rlxds7ssq7nk2in4s46xws7xy9ylxsqgcz85hxjgh17lsm0y39c";
+    url = "https://github.com/rafael2k/darkice/releases/download/v${version}/darkice-${version}.tar.gz";
```

Comment:
Why are we not fetching the source archive?

Reply:
The repository is obviously a direct copy of the old SVN repo, this seems to be the better option.

Reply:
oh yeah, I didn't check that the export is like this.